charts/port-ocean/templates/cron-job/installation-resync-job.yml

```diff
@@ -1,7 +1,7 @@
 {{- $resyncOnInstall := (.Values.workload.cron).resyncOnInstall | default true }}
 {{- $resyncOnUpgrade := (.Values.workload.cron).resyncOnUpgrade | default true }}
 {{- if and (eq .Values.workload.kind "CronJob") (or $resyncOnInstall $resyncOnUpgrade) }}
-{{- $jobName := .Release.Revision }}
+{{- $jobName := print (randAlphaNum 4) .Release.Revision }}
```

Comment:
Don't we want some separator between the random string and the revision?

Reply:
fair